sdk/integration-test/integration-testkit-test.js

```diff
@@ -29,14 +29,51 @@ action.commandHandlers = {
   }
 };
 
+const value_entity = new akkaserverless.ValueEntity("./test/example.proto", "com.example.ExampleServiceTwo", "value-entity-example-service");
+
+value_entity.setInitial(entityId => (value_entity.lookupType("com.example.Example").create({})));
+
+value_entity.commandHandlers = {
+  DoSomethingOne: input => {
+    return {field: "ValueEntity Received " + input.field};
+  },
+  DoSomethingTwo: input => {
+    return new Promise((resolve) => {
+      setTimeout(() => resolve({field: "ValueEntityAsync Received " + input.field}), 1000);
+    });
+  }
+};
+
+const entity = new akkaserverless.EventSourcedEntity("./test/example.proto", "com.example.ExampleServiceThree", "event-sourced-entity-example-service");
+
+entity.setInitial(entityId => (entity.lookupType("com.example.Example").create({})));
+
+entity.setBehavior(e => {
+  return {
+    commandHandlers: {
+      DoSomethingOne: input => {
+        return {field: "EventSourcedEntity Received " + input.field};
+      },
+      DoSomethingTwo: input => {
+        return new Promise((resolve) => {
+          setTimeout(() => resolve({field: "EventSourcedEntityAsync Received " + input.field}), 1000);
+        });
```

Comment:
One thought, not sure we need to deal with it in this PR: One thing that could be nice to align with the rest of the api would be to have a convenience factory `Reply.promise`, since all kinds of other ways of replying and side effects etc is done that way. By having that a user can discover what is possible to return by auto completing on `Reply.` (in addition to looking at the JSdoc).

Perhaps a bit weird that you'd want to be able to also resolve the promise itself with `Reply` though `Reply.message().withSideEffects().whatNot()`

Reply:
If you agree we can follow up on this good suggestion in a separate PR.
I think I would prefer to have first a consistent API experience for entities / value-entities etc.

Reply:
Yeah, lets defer that and do it separately.